zcbor/zcbor.py

```diff
@@ -1735,11 +1735,15 @@ def from_cddl(cddl_class, mode, *args, **kwargs):
 
         return cddl_res
 
+    # Whether this element (an OTHER) refers to an entry type.
+    def is_entry_type(self):
+        return (self.type == "OTHER") and (self.value in self.entry_type_names)
+
     # Whether to include a "cbor" variable for this element.
     def is_cbor(self):
-        return (self.type_name() is not None) and (
-            (self.type != "OTHER")
-            or ((self.value not in self.entry_type_names) and self.my_types[self.value].is_cbor()))
+        res = (self.type_name() is not None) and not self.is_entry_type() and (
+            (self.type != "OTHER") or self.my_types[self.value].is_cbor())
+        return res
```

Comment:
Any specific reason to also change the syntax here? Just readability?

Reply:
Hmm I don't remember. It seems to be equivalent. I extracted part of it into the `is_entry_type()`. The use of `res` is likely a remnant from me wanting to print the result at one point while troubleshooting :).